apps/cookbook/src/app/page/header/header.component.scss

```
@@ -39,6 +39,12 @@ nav {
     padding: 0;
     margin: 0;
 
+    li::before {
+      display: none;
+    }
+    li + li {
+      margin-block-start: revert;
+    }
```

Minor detail: add line break between rulesets
```suggestion
    }
    
```